### ISSUE TITLE
Add Bedrock Support

### ISSUE THREAD

**Issue by ItsDrike** [py-mine/mcproto#3]
This library currently only supports the Java version of minecraft, add support for bedrock.

**Comment by ItsDrike** [comment]
This feature is currently not planned and is considered as deferred. It is NOT available to be worked on by others, at least not until version 1.0.0 is released, after which this might make more sense for the library.

This is because even if someone would be willing to write a working implementation for bedrock, the project doesn't currently have maintainers willing to keep this implementation up to date and fix potential bugs in it. 

It would also likely require a pretty big restructurization of the project, as the current file structure is set up to just cover the java implementation.

**Comment by ANDREI12333** [comment]
Whenever this is being worked on. I'd recommend splitting the project files. Basicly the java files into a **java** folder. And the bedrock ones being split into a **be** folder.

**Comment by ANDREI12333** [comment]
Yes. But due to the split. Imports would be `quarry.java` or `quarry.be` instead of `quarry` (The module name)

**Comment by ItsDrike** [comment]
After some consideration, I've decided that bedrock support would not make sense for this project, very likely not even in the long term. It would require a lot of new code, and essentially a complete split between the java version and bedrock, as these are completely incompatible.

I'd prefer this be left to another library to handle. This library might still benefit from   utilizing the buffer and connection classes that mcproto already provides, and extending them, so mcproto might become a dependency of that library, but mcproto itself will not support bedrock any time soon, certainly not even after 1.0.0, as my previous comment suggested it might.

There is some slight potential for this issue to get re-opened at some point, if bedrock support really would be something we'd want to do, however I still think that should be a task left for another library (it could however potentially one under the py-mine banner).
